rs/nns/governance/src/neuron_store.rs

```diff
@@ -807,14 +807,14 @@ impl NeuronStore {
     }
     pub fn with_active_neurons_iter<R>(
         &self,
-        callback: impl for<'b> FnOnce(Box<dyn Iterator<Item = Neuron> + 'b>) -> R,
+        callback: impl for<'b> FnOnce(Box<dyn Iterator<Item = Cow<Neuron>> + 'b>) -> R,
```

Comment:
NCR;

this type is getting pretty involved; consider specifying it using `with` if that can make it more readable.